# SPDX-FileCopyrightText: Huawei Inc.
#
# SPDX-License-Identifier: Apache-2.0

Patch for //third_party/libevdev git repository of OpenHarmony 3.1 codebase.

Signed-off-by: Esben Haabendal <esben.haabendal@huawei.com>
Upstream-Status: Inappropriate [configuration/integration]

diff --git a/include/linux/linux/input.h b/include/linux/linux/input.h
index 537d23e..e58f84c 100644
--- a/third_party/libevdev/include/linux/linux/input.h
+++ b/third_party/libevdev/include/linux/linux/input.h
@@ -15,7 +15,7 @@
 #include <sys/types.h>
 #include <linux/types.h>
 
-#include "input-event-codes.h"
+#include <linux/input-event-codes.h>
 
 /*
  * The event structure itself
